packages/ember-data/lib/system/model/model.js

```diff
@@ -683,10 +683,7 @@ var Model = Ember.Object.extend(Ember.Evented, {
     successfully or rejected if the adapter returns with an error.
   */
   save: function() {
-    var model = this;
-    return this.reference.save().then(function() {
-      return model;
-    });
+    return this.reference.save();
```

Comment:
reload wraps in the record file, so if we want to be consistent, maybe makes more sense to wrap here? Reference deals with internals, and model.js deals with exposing itself to the world?